docs/source/design/control_plane.rst

```diff
@@ -317,6 +317,42 @@ To fetch output of a specific node execution:
 
 :ref:`Node <flyte:divedeep-nodes>` here, can correspond to a task, workflow, or branch node.
 
+Reference launch plan executions
+====
```

Comment:
This needs to be as long as the title above -- I think that should clear up the build warning!